From 2c7133b4de8f44d416b393737a36dba8f6d8a6b1 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:26:03 -0400
Subject: [alsa] hda: remove unused clear of STATESTS

Message-id: <140972823573-206-patchset-jkysela@redhat.com>
Patchwork-id: 90906
O-Subject: [RHEL 7.1 PATCH 205/442] ALSA: hda - remove unused clear of STATESTS
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

Although the code was updated last year the "#if 0" surrounding it
dates back to the original git commit.  The function will be moved to
a new file, no need to carry the dead code.

Signed-off-by: Dylan Reid <dgreid@chromium.org>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: f563bf65d9028c58bb0ecea9c8d40d5a57e64b3f)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/hda/hda_intel.c b/sound/pci/hda/hda_intel.c
index 0e879e1..079e5a4 100644
--- a/sound/pci/hda/hda_intel.c
+++ b/sound/pci/hda/hda_intel.c
@@ -1081,11 +1081,6 @@ static irqreturn_t azx_interrupt(int irq, void *dev_id)
   azx_writeb(chip, RIRBSTS, RIRB_INT_MASK);
  }
 
-#if 0
- /* clear state status int */
- if (azx_readw(chip, STATESTS) & 0x04)
-  azx_writew(chip, STATESTS, 0x04);
-#endif
  spin_unlock(&chip->reg_lock);
  
  return IRQ_HANDLED;
-- 
1.7.1